x-pack/plugins/ml/public/application/explorer/alerts/alerts_summary.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBadge, EuiDescriptionList, EuiFlexGroup, EuiFlexItem, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { ALERT_DURATION, ALERT_END } from '@kbn/rule-data-utils';
+import React, { useMemo } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { getAlertFormatters } from '../../../alerting/anomaly_detection_alerts_table/render_cell_value';
+import { useMlKibana } from '../../contexts/kibana';
+import { useAnomalyExplorerContext } from '../anomaly_explorer_context';
+import { getAlertsSummary } from './get_alerts_summary';
+
+export const AlertsSummary: React.FC = () => {
+  const {
+    services: { fieldFormats },
+  } = useMlKibana();
+  const { anomalyDetectionAlertsStateService } = useAnomalyExplorerContext();
+
+  const alertsData = useObservable(anomalyDetectionAlertsStateService.anomalyDetectionAlerts$, []);
+  const formatter = getAlertFormatters(fieldFormats);
+
+  const sortedAlertsByRule = useMemo(() => {
+    return getAlertsSummary(alertsData);
+  }, [alertsData]);
+
+  return (
+    <EuiFlexGroup>
+      {sortedAlertsByRule.map(([ruleName, ruleSummary]) => {
```

Comment:
The layout needs some work here as it currently isn't working well when there are more than a few rules:

<img width="752" alt="image" src="https://github.com/elastic/kibana/assets/7405507/980c67fb-ed0a-473b-a9a2-c575d4611cfb">

Reply:
updated with the flex grid using 3 column in https://github.com/elastic/kibana/pull/167998/commits/9ef5bc56e5d3c6b106f559a754e61f790a5e8f65

Reply:
@darnautov what would happen if there were e.g. 100 rules? Is there paging in the summary panel?

Reply:
Added pagination in 836273732fe5af88c2947678750744b8085f867e

Reply:
I think we can improve this part (rules) of the summary view.  But I think it makes sense to do this in a specific issue/PR.